core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/kerberos/CmServiceKeytabRequestFactory.java

```diff
@@ -27,7 +27,7 @@ public ServiceKeytabRequest create(Stack stack, GatewayConfig primaryGatewayConf
         request.setDoNotRecreateKeytab(Boolean.TRUE);
         RoleRequest roleRequest = new RoleRequest();
         roleRequest.setRoleName("hadoopadminrole-" + stack.getName());
-        roleRequest.setPrivileges(Set.of("Service Administrators", "Certificate Administrators", "CA Administrator"));
+        roleRequest.setPrivileges(Set.of("Service Administrators", "Certificate Administrators", "Host Administrators", "CA Administrator"));
```

Comment:
What does this do?

Reply:
When we configure load-balancer in Oozie, CM tries to add the host in FreeIPA. It was failing because it did not have permission to add the role for the host.